sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -431,7 +431,7 @@ object functions {
    * @since 2.0.0
    */
   def first(e: Column, ignoreNulls: Boolean): Column = withAggregateFunction {
-    new First(e.expr, Literal(ignoreNulls))
+    new First("first", e.expr, Literal(ignoreNulls))
```

Comment:
We can add a constructor to provide a default name.

Reply:
Done